omnipaxos/src/omni_paxos.rs

```diff
@@ -394,6 +394,13 @@ where
         }
     }
 
+    /// Manually attempt to become the leader with the round number `n`.
+    pub fn become_leader(&mut self, n: u32) {
+        let mut my_ballot = self.ble.get_current_ballot();
```

Comment:
I think we should add a check that the given n is higher than our current ballot, return `Err` if it's not.